src/services/game.online.service.mjs

```diff
@@ -96,3 +95,25 @@ export const setGameState = async (gameId, state) => {
     throw new Error("Couldn't set state for game/" + gameId);
   }
 };
+
+export const getScoreboard = (gameData) => {
+  if (!gameData || !gameData?.scores || !gameData?.players)
+    throw new Error("Invalid gameData");
+  const players = Object.entries(gameData.players);
+  const scores = gameData.scores;
```

Comment:
Das feld heißt score in der DB. aber ich stimme dir zu, sollten wir scores nennen. Müssen wir projektweit ändern